### PR TITLE
Add support for the Debian derivative TurnKey Linux

### PR DESCRIPTION
**Pull request by dafyddj**
### What does this PR do?
Adds support for the Debian derivative TurnKey Linux (v15, based on Debian Stretch).
https://www.turnkeylinux.org/

### What issues does this PR fix or reference?
Very similar to #1299 and PR #1300

### Previous Behavior
```
Stdout from the command:

 *  INFO: Running version: 2019.01.08
 *  INFO: Executed by: 
 *  INFO: Command line: 'bootstrap-salt.sh -F -c /tmp -A 127.0.0.1 -i minion -F -c /tmp'

 *  INFO: System Information:
 *  INFO:   CPU:          GenuineIntel
 *  INFO:   CPU Arch:     x86_64
 *  INFO:   OS Name:      Linux
 *  INFO:   OS Version:   4.9.0-8-amd64
 *  INFO:   Distribution: TurnKey 9.7

 *  INFO: Installing minion
 *  INFO: Found function config_salt
 *  INFO: Found function preseed_master
 *  INFO: Found function daemons_running
 * ERROR: No dependencies installation function found. Exiting...
```

### New Behavior
Installs Salt correctly.

### CODE REVIEW

File: bootstrap-salt.sh [hunk 1]
```diff
@@ -1472,6 +1473,10 @@ __debian_derivatives_translation() {
                 _major=$(echo "$DISTRO_VERSION" | sed 's/^\([0-9]*\).*/\1/g')
                 _debian_derivative="bunsenlabs"
                 ;;
+            turnkey)
+                _major=$(echo "$DISTRO_VERSION" | sed 's/^\([0-9]*\).*/\1/g')
+                _debian_derivative="bunsenlabs"
```
Comment (vutny) [inline bootstrap-salt.sh]: @dafyddj Copy paste?

Comment (dafyddj) [reply bootstrap-salt.sh]: Yes, you're right - my mistake.

**Comment by dubb-b** [comment]
@dafyddj messing with the build system. I am going to re-trigger the builds.

**Review by vutny — APPROVED** [review]

**Comment by vutny** [comment]
The build has passed, just need to update the branch with `develop`.

**Pull request merged**